features/machine/machine.feature

```diff
@@ -114,3 +114,32 @@ Feature: Machine features testing
     Given I have an IPI deployment
     And evaluation of `BushSlicer::Machine.list(user: admin, project: project('openshift-machine-api'))` is stored in the :machines clipboard
     Then the expression should be true> cb.machines.select{|m|m.instance_state == m.annotation_instance_state}.count == cb.machines.count
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-27609
+  @admin
+  @destructive
+  Scenario: Scaling a machineset with providerSpec.publicIp set to true
+    Given I have an IPI deployment
+    And I clone a machineset named "machineset-clone-publiciptrue"
+
+    Then I run the :get admin command with:
+     | resource       | machineset                       |
+     | resource_name  | machineset-clone-publiciptrue    |
+     | namespace      | openshift-machine-api            |
+     | o              | yaml                             |
```

Comment:
I think people are asking to remove extra spaces and align them again to make a compact view.

```
| resource_name  | machineset-clone-publiciptrue    |
```
to
```
| resource_name | machineset-clone-publiciptrue |
```

Reply:
ok got it , Thanks ! made changes as per your comments